### ISSUE TITLE
Add Database Schema Support

### ISSUE THREAD

**Issue by riipandi** [authorizerdev/authorizer#178]
It will be nice to have if the Authorizer supports the database schema.

This will be useful when using a shared database such as PostgreSQL in one instance.

Since Authorizer using Gorm this will not too hard to implement.

**Comment by lakhansamani** [comment]
@riipandi yes. That is true but now after so many releases I will have to think about backward compatibility else if tables changes one might not get the required data.
